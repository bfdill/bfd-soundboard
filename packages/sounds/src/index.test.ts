describe('sounds', () => {
  test('has known exports', () => {
    expect(Object.keys(require('.'))).toMatchSnapshot()
  })
})
