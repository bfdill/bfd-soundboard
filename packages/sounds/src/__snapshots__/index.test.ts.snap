// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sounds has known exports 1`] = `
Array [
  "SoundService",
  "LOGGER_META",
]
`;
